js/app/utils.js

```diff
@@ -584,3 +584,11 @@ function parse_uri (uri_str, parse_query) {
 
     return {scheme, user, password, host, port, path, query, fragment};
 }
+
+function open_app_center_details (app_id) {
```

Comment:
Do you think this will be used in other place?

Reply:
nope, as we discussed in slack, I will move it back to the Dialog.